u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessResponse.java

```diff
@@ -0,0 +1,104 @@
+package com.google.u2f.key.messages;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import com.google.common.base.Preconditions;
+import com.google.u2f.server.ControlFlags;
+
+/**
+ * TransferAccessResponse is constructed by a device to whom access has been transferred. It wraps 
+ * the TransferAccessMessage chain with necessary information to register a new key.
+ */
+public class TransferAccessResponse extends AuthenticateResponse {
+  private final byte controlFlags;
+  private final TransferAccessMessage[] transferAccessMessages;
+  private final byte[] keyHandle;
+  private final int counter;
+  private final byte[] signature;
+
+  /**
+   * Constructor for the TransferAccessResponse
+   * 
+   * @param controlFlags: See ControlFlags. Should indicate this is a TransferAccessResponse
```

Comment:
see controlflags?

A better message might be `@paramcontrolFlags sets the {@link ControlFlags} for this response.